File: Android/app/src/main/java/io/agora/chatroom/model/Constant.java
package io.agora.chatroom.model;

import android.text.TextUtils;

import io.agora.chatroom.util.MemberUtil;

public class Constant {

    /**
     * Here give a random userId, You can use your business userId
     */
    public static final int sUserId = MemberUtil.getUserId();

    public static boolean isMyself(String userId) {
        return TextUtils.equals(userId, String.valueOf(Constant.sUserId));
    }

    public static final String sName = MemberUtil.getName();
    public static final int sAvatarIndex = MemberUtil.getAvatarIndex();

}

File: iOS/AgoraChatRoom/Constant.swift
//
//  Constant.swift
//  AgoraChatRoom
//
//  Created by LXH on 2019/11/27.
//  Copyright © 2019 CavanSu. All rights reserved.
//

import Foundation

struct Constant {
    static let sUserId: UInt = UInt(UInt32(bitPattern: MemberUtil.getUserId()))

    static func isMyself(_ userId: String) -> Bool {
        userId == String(sUserId)
    }

    static let sName = MemberUtil.getName()
    static let sAvatarIndex = MemberUtil.getAvatarIndex()
}
